src/js/profile/wearable/widget/wearable/SnapListview.js

```diff
@@ -683,6 +683,7 @@
 				self._selectedIndex = null;
 				self._currentIndex = null;
 
+				scrolling.scrollToIndex(0, 0);
```

Comment:
I'm not sure this change. This patch causes unwanted effects during the transition animation between pages. User can see items from top of page during page transition.

Reply:
If you change to a different page Isn't it common to go to the top of the page?
Could you give me an example of a case you are worried about?

Reply:
Hi,
I prepared example for check your PR. (https://github.com/Samsung/TAU/pull/524)
You can test your commit in web browser with slow animations.
![image](https://user-images.githubusercontent.com/29534410/71574094-81b29d80-2ae7-11ea-895e-f7f9a2e5d23e.png)

Reply:
Thank you to share the example.
I will check it!